spec/features/repositories_spec.rb

```diff
@@ -63,9 +73,76 @@
       fill_in 'search-input-field', with: 'python'
       find('#filter-button').click
 
-      expect(page).to have_text(repository.languages) &
-                      have_text(second_repository.languages)
-      expect(page).to have_css('.list-group .list-group-item', :count => 2)
+      expect(page).to have_text('python')
+      expect(page).to have_css('#repository-card', count: 2)
+    end
+  end
+
+  context 'when the repository has no issues and stars' do
+    let!(:repository) { create(:repository).decorate }
+
+    before do
+      visit repositories_path
+    end
+
+    it 'have all issues for repositories' do
+      expect(page).not_to have_css('#issues-tag')
+    end
+
+    it 'have all stars for repositories' do
+      expect(page).not_to have_css('#stars-tag')
+    end
+  end
+
+  context 'when the repository has issues' do
+    context 'when issues are greater than 1000' do
+      let!(:repository) { create(:repository, issues: '5455').decorate }
+
+      before do
```

Comment:
I think that you could put this code bloc between lines 97 and 98, to avoid repeat every time

Reply:
If I try to move this code bloc to the upper context, tests will fail. Because I will define this code bloc before to define the repository